src/cmd-build

```diff
@@ -172,16 +179,37 @@ touch .build-commit
 mv -T ${tmp_builddir} ${buildid}
 # Replace the latest link
 ln -Tsfr "${buildid}" latest
-python3 -c 'import os,json,sys,stat;
+# Regenerate builds.json
+python3 -c "import os,json,sys,stat,datetime;
 builds=[]
-for d in os.listdir("."):
+for d in os.listdir('.'):
   s = os.lstat(d)
-  if not (stat.S_ISDIR(s.st_mode) and os.path.isfile(d + "/meta.json")):
+  meta_json = d + '/meta.json'
+  if not (stat.S_ISDIR(s.st_mode) and os.path.isfile(meta_json)):
     continue
-  builds.append((d, s.st_ctime))
-json.dump({"builds":
-           list(map(lambda x: x[0], sorted(builds, key=lambda x: x[0])))},
-          sys.stdout)
-' > ${workdir}/tmp/builds.json
+  j = {}
+  with open(meta_json) as f:
+    j = json.load(f)
+  t = datetime.datetime.strptime(j['ostree-timestamp'], '%Y-%m-%dT%H:%M:%SZ')
+  builds.append((d, t))
+builds = sorted(builds, key=lambda x: x[1], reverse=True)
```

Comment:
This also fixes a slight bug in the sorting: we were previously sorting by the first key (the directory name) instead of the directory creation time. (We actually change the strategy here to just use the `ostree-timestamp` instead.)